testutils/utils.go

```diff
@@ -75,7 +75,7 @@ func GenerateEthTxs(t *testing.T) (*gethtypes.Transaction, *gethtypes.Transactio
 	require.NoError(t, err)
 	l1InfoTx := gethtypes.NewTx(l1InfoRawTx)
 
-	rng := rand.New(rand.NewSource(1234))
+	rng := rand.New(rand.NewSource(12345))
```

Comment:
Instead of modifying the seed could you just set the `Mint` field directly in the test helper? We'll also want to test the case where `Mint` is nil
You should be able to modify it [here](https://github.com/polymerdao/monomer/blob/ddcdad2bd95fd01645f7225f8ab770569b56741a/testutils/utils.go#L79) before calling `gethtypes.NewTx`

Reply:
Alternatively, should we just use a random seed for every test so we can check that our tests pass for all deposit tx variations?

cc @joshklop

Reply:
Do you want me to implement a test for the case when `Mint` is nil in this or another PR?

Reply:
Hmm actually for now let's just leave `Mint` as always set and then we should open up a new issue to ensure that we have some fuzz testing for all possible deposit tx types (with and without a mint field, with and without a value field, etc.). It probably doesn't make sense to fuzz test all of our unit/integration tests but it's definitely good to keep in mind